README.md

```diff
@@ -75,7 +83,14 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+// var pizza={
+temperature:70;
+function bake(){
+  this.temperatue=300;
+};
+}
```

Comment:
Watch your syntax on this. [Check the solution.](https://github.com/ga-wdi-quizzes/week-02/blob/solution/README.md#question-6)